### ISSUE TITLE
recover from the backup file noe work

### ISSUE THREAD

**Issue by geekyouth** [twofas/2fas-android#62]
I have backup my 2fsAuth APP by Android BackUp APP ,but when i recover from the backup file , 2FS Auth APP can not work ,boot then exit.

![image](https://github.com/twofas/2fas-android/assets/12899262/21e97fb2-067f-4232-b839-6a34fc778873)

**Comment by rafakob** [comment]
It is not possible to recover data from backup files created by a third-party application. For security measures, the 2FAS local database is encrypted with a random key generated during app installation (each app installation has a different key). This key is securely stored in the system keystore and cannot be accessed outside the scope of the 2FAS app. That's why we offer other backup methods - manual or automatic through Google Drive.

**Comment by geekyouth** [comment]
I don't know what to do now. I even cloned the source code of the 2fas-android project and opened it with Android Studio. I found the code related to 'import from file', but I still don't know how to import my backup data: 

database-2fas-shm, database-2fas-wal, and database-2fas.

**Comment by rafakob** [comment]
"Import from file" option is for backup files created using our in-app solution in "Settings -> 2FAS Backup -> Export to file".

Those database files are encrypted and it's impossible to import them. It's done for security measures.

**Comment by geekyouth** [comment]
thank you , now i have to give up recovery my  backup file from android system backup app ???

**Comment by rafaelazvdo** [comment]
> thank you , now i have to give up recovery my  backup file from android system backup app ???
> 

Yeah, Android backup is completely useless for this kind of apps.
